xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -1206,6 +1217,90 @@ struct AppModel: ModelProtocol {
                 state: state,
                 environment: environment
             )
+        case .succeedMoveEntry(from: let from, to: let to):
+            return Update(state: state)
+        case .succeedMergeEntry(parent: let parent, child: let child):
+            return Update(state: state)
+        case .succeedSaveEntry(address: let address, modified: let modified):
+            return Update(state: state)
+        case .succeedUpdateAudience(_):
+            return Update(state: state)
+        case let .saveEntry(entry):
+            return saveEntry(
+                state: state,
+                environment: environment,
+                entry: entry
+            )
+        case let .moveEntry(from, to):
+            return moveEntry(
+                state: state,
+                environment: environment,
+                from: from,
+                to: to
+            )
+        case let .mergeEntry(parent, child):
+            return mergeEntry(
+                state: state,
+                environment: environment,
+                parent: parent,
+                child: child
+            )
+        case let .updateAudience(address, audience):
+            return updateAudience(
+                state: state,
+                environment: environment,
+                address: address,
+                audience: audience
+            )
+        case .failSaveEntry(address: let address, error: let error):
+            logger.warning("""
+                Failed to save entry: \(address)
+                \(error)
+                """)
```

Comment:
Nit: indendation. Swift multiline strings are sensitive to the indentation of opening and closing quotes. They need to match text indentation. Should be:

```swift
logger.warning(
    """
    Failed to save entry: \(address)
    \(error)
    """
)
```